File: src/main/java/mk/jug/collectors/model/Player.java
package mk.jug.collectors.model;

/**
 * NBA Player
 */
public class Player {
    private final String name;
    private final Integer points;

    public Player(String name, Integer points) {
        this.name = name;
        this.points = points;
    }

    public String getName() {
        return name;
    }

    public Integer getPoints() {
        return points;
    }

    public static Player fromString(String line) {
        String[] parts = line.split(",");
        return new Player(parts[0], Integer.parseInt(parts[1]));
    }

    @Override
    public String toString() {
        return String.format("%-30s%5d", name, points);
    }
}

File: src/main/kotlin/mk/jug/collectors/model/Player.kt
package mk.jug.collectors.model

/**
 * NBA Player
 */
class Player(val name: String, val points: Int) {

    override fun toString(): String {
        return String.format("%-30s%5d", name, points)
    }

    companion object {
        fun fromString(line: String): Player {
            val parts = line.split(",")
            return Player(parts[0], Integer.parseInt(parts[1]))
        }
    }
}
